core/src/main/java/com/linecorp/armeria/common/metric/MoreMeters.java

```diff
@@ -52,9 +106,37 @@ public static DistributionSummary summaryWithDefaultQuantiles(MeterRegistry regi
                                   .register(registry);
     }
 
+    /**
+     * Returns a newly-registered {@link Timer} configured by {@link MoreMeters#distributionStatisticConfig}.
```

Comment:
`{@link #distributionStatisticConfig()}`

- We should not refer to a field but to a method.
- `MoreMeters` is redundant.